packages/drivechain_client/android/.gitignore

```diff
@@ -0,0 +1,13 @@
+gradle-wrapper.jar
+/.gradle
+/captures/
```

Comment:
Might as well nuke the `android` and `ios` folders?

Reply:
Yes, not needed here